### PR TITLE
Fixed typos in comment

### PR DESCRIPTION
**Pull request by Ullallulloo**
Fixed some typos in comments and removed redundant function call in checking if space or tab

### CODE REVIEW

File: lib/rules_block/list.js [hunk 1]
```diff
@@ -210,12 +210,10 @@ module.exports = function list(state, startLine, endLine, silent) {
     while (pos < max) {
       ch = state.src.charCodeAt(pos);
 
-      if (isSpace(ch)) {
-        if (ch === 0x09) {
-          offset += 4 - (offset + state.bsCount[nextLine]) % 4;
-        } else {
-          offset++;
-        }
+      if (ch === 0x09) {
```
Comment (puzrin) [inline lib/rules_block/list.js]: I would leave this block change intact, to better understand relation to spec.

**Comment by Ullallulloo** [comment]
Okay. Sorry, I changed that back now.

**Comment by puzrin** [comment]
Please join into single commit to keep history clean :)

**Comment by Ullallulloo** [comment]
Um, okay. Is that right?

**Comment by puzrin** [comment]
\+ 3 more commits :) 

Try this now:

```sh
git reset HEAD~5
git commit -a -m 'Fixed typos in comment'
git push -f
```

**Comment by Ullallulloo** [comment]
Oh dear, yes. I think I did it now at least. Sorry the bother.

**Comment by puzrin** [comment]
Thanks for fixes and patience :)

**Comment by Ullallulloo** [comment]
Lol, thank you. :)

**Pull request merged**